readme.md

```diff
@@ -97,6 +98,7 @@ Currently, these are the flags you can configure:
 - `tests` - Run `npm test` (`true` by default).
 - `yolo` - Skip cleanup and testing (`false` by default).
 - `publish` - Publish (`true` by default).
+- `preview` - Only show what np is going to execute without running anything (`false` by default).
```

Comment:
This should be added to the `Config` section as well.

Reply:
But... This has already been done

Reply:
Sorry, I missed that for some reason.